frontend/src/lib/utils.ts

```diff
@@ -4,3 +4,7 @@ import { twMerge } from "tailwind-merge";
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+
+export function title({ title }: { title: string }) {
+  return `${title} | ZaneOps`;
+}
```

Comment:
```suggestion
export function MetaTitle({ title }: { title: string }) {
  return <title>{`${title} | ZaneOps`}</title>;
}
```

Make this into a component `meta-title` (and put inside of its own file)